src/components/AutocompleteField/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import fieldIsValid from 'form/fieldIsValid';
+import fieldIsInvalid from 'form/fieldIsInvalid';
+import FormLabel from 'components/FormLabel';
+import FormValidationMessage from 'components/FormValidationMessage';
+import FormValidationIcon from 'components/FormValidationIcon';
+import styles from './autocomplete-field.scss';
+import Autocomplete from 'containers/Autocomplete';
+
+export default ({ config, helper }) => (
+  <div className={config.hidden ? styles.hidden : styles.root}>
+    {!config.hidden &&
+      <FormLabel
+        fieldId={config.name}
+        label={config.label}
+        hint={config.hint}
+      />
+    }
+
+    <div className={styles.wrapper}>
+      <Autocomplete
+        helper={helper}
+        {...config}
+      />
+
+      {!config.hidden &&
+        <div>
+          <div className={styles.message}>
+            <FormValidationMessage
+              error={fieldIsInvalid(helper)}
+              valid={fieldIsValid(helper)}
+              message={helper.error || config.validated}
+            />
+          </div>
+
+          <div className={styles.icon}>
+            <FormValidationIcon
+              error={fieldIsInvalid(helper)}
+              valid={fieldIsValid(helper)}
+            />
+          </div>
+        </div>
+      }
+    </div>
+  </div>
+);
```

Comment:
Hmm, I'm not sure what to suggest. Could we make this a wrapping / higher order component that accepts a child and share it between input and autocomplete?

So

```
<div className={styles.wrapper}>
    {this.props.children}
```

instead of

```
<div className={styles.wrapper}>
  <Autocomplete
    helper={helper}
    {...config}
  />
```